### DISCUSSION TITLE
Enable webpack5 persistent cache to pack more slowly for the first time

### DISCUSSION THREAD

**Discussion by fandywen** [webpack/webpack#16012] [General]
![image](https://user-images.githubusercontent.com/22906355/176841720-e670ab67-9ead-4bfe-a196-f1b219526630.png)

**Comment by vankop** [comment]
Thank you for creating this issue. However, issues need to follow one of our templates so that we can clearly understand your particular circumstances.

Please help us by recreating the issue using one of our templates.

Please fill out all fields, we can't help you without it

**Comment by fandywen** [comment]
I want to know why the first packaging and construction speed is slower after webpack5 persistent cache is enabled
Webpack5 persistent cache code can be optimized
See the above figure

**Comment by fandywen** [comment]
> Thank you for creating this issue. However, issues need to follow one of our templates so that we can clearly understand your particular circumstances.
> 
> Please help us by recreating the issue using one of our templates.
> 
> Please fill out all fields, we can't help you without it

Why Before introducing persistent cache 3.221s
After introducing persistent cache  3.666s

**Reply by alexander-akait** [reply]
Cache writting on disk requires time
